### DISCUSSION TITLE
支持以虚拟目录的形式挂载其他人分享的文件夹

### DISCUSSION THREAD

**Discussion by karma-sun** [messense/aliyundrive-webdav#133] [Ideas]
例如指定一个分享链接https://www.aliyundrive.com/s/xxx
再指定一个虚拟目录vdir，则访问/vdir即可访问分享链接https://www.aliyundrive.com/s/xxx的所有文件

**Comment by messense** [comment]
建议直接转存。

**Comment by xausky** [comment]
直接转存有两个弊端，一个是占用自身网盘容量，另一个是在使用一些连载资源时无法同步更新。这个需求我最近看看有能力的话可以提个PR。
